adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -54,6 +54,7 @@
 # pylint: disable=bad-whitespace
 _SET_NET_CMD           = const(0x10)
 _SET_PASSPHRASE_CMD    = const(0x11)
+_SET_AP_PASSPHRASE_CMD = const(0x19)
```

Comment:
woops, this is copy paste from my AP pull request. This can be removed from this PR.